roles/contrail_node/meta/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+galaxy_info:
+  author: Savithru Lokanath
+  description: Contrail Networking
+  company: Juniper Networks, Inc.
+  license: Apache License, Version 2.0
+  min_ansible_version: 2.2
+  platforms:
+  - name: EL
+    versions:
+    - 7
+  categories:
+  - cloud
+  - system
+dependencies:
+- role: openshift_facts
+- role: openshift_master_facts
```

Comment:
Why master facts? Not node facts?

Reply:
@ingvagabund will `openshift-node` playbook invoke node facts?